requirements.txt

```diff
@@ -12,3 +12,4 @@ PyYAML==5.3.1
 ansicolors==1.1.8
 pyusb==1.1.0
 six>=1.13.0
+https://github.com/Yepkit/pykush/archive/master.zip#egg=pykush
```

Comment:
As I read this you are not using a tag from the project but resent master. In order to maintain CI stability and to avoid sudden regressions due to dependencies breaking something I believe you must point to a certain version.